crates/bevy_solari/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "bevy_solari"
```

Comment:
I've given this some thought and I'm reasonably convinced that using the Bevy Solari branding here is useful:

* It allows us to more accurately discuss this specific implementation relative to raytracing in general
* It gives us a name we can hype up
* It gives future competing implementations the ability to do the same
* We can still use more "technically descriptive" names for the types people will most commonly interact with (ex: the names of components)